<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Minimal Podcasts Player</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/img/icon.svg</normaloff>:/img/icon.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="playerLayout" stretch="1,0,0,0,1,0,0,0,0,0,0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>6</number>
        </property>
        <property name="topMargin">
         <number>6</number>
        </property>
        <property name="rightMargin">
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="curPCLabel">
            <property name="text">
             <string>Podcast</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="curTrackName">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Episode Title</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="queuePrevBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="media-skip-backward">
            <normaloff>../../../../../../.designer/backup</normaloff>../../../../../../.designer/backup</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="back10Btn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="media-seek-backward">
            <normaloff>../../../../../../.designer/backup</normaloff>../../../../../../.designer/backup</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="timeLabel">
          <property name="text">
           <string notr="true">0:00:00</string>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="timeSlider">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="totalTimeLabel">
          <property name="text">
           <string notr="true">0:00:00</string>
          </property>
          <property name="margin">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="playBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="media-playback-start">
            <normaloff>../../../../../../.designer/backup</normaloff>../../../../../../.designer/backup</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stopBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="media-playback-stop">
            <normaloff>../../../../../../.designer/backup</normaloff>../../../../../../.designer/backup</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="for10Btn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="media-seek-forward">
            <normaloff>../../../../../../.designer/backup</normaloff>../../../../../../.designer/backup</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="queueNextBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="media-skip-forward">
            <normaloff>../../../../../../.designer/backup</normaloff>../../../../../../.designer/backup</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="optionsBtn">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="applications-system">
            <normaloff>../../../../../../.designer/backup</normaloff>../../../../../../.designer/backup</iconset>
          </property>
          <property name="popupMode">
           <enum>QToolButton::MenuButtonPopup</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="volumeBtn">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset theme="audio-volume-high">
            <normaloff>../../../../../../.designer/backup</normaloff>../../../../../../.designer/backup</iconset>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Home</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QWidget" name="widget" native="true">
            <layout class="QGridLayout" name="gridLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QSplitter" name="splitter">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QWidget" name="widget_2" native="true">
                <layout class="QGridLayout" name="gridLayout_5">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
                   <item>
                    <widget class="QListWidget" name="podcastsList">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="minimumSize">
                      <size>
                       <width>240</width>
                       <height>0</height>
                      </size>
                     </property>
                     <property name="contextMenuPolicy">
                      <enum>Qt::CustomContextMenu</enum>
                     </property>
                     <property name="editTriggers">
                      <set>QAbstractItemView::NoEditTriggers</set>
                     </property>
                     <property name="selectionMode">
                      <enum>QAbstractItemView::SingleSelection</enum>
                     </property>
                     <property name="selectionBehavior">
                      <enum>QAbstractItemView::SelectRows</enum>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="widget_3" native="true">
                <layout class="QGridLayout" name="gridLayout_6" rowstretch="0,1">
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QWidget" name="widget_5" native="true">
                   <property name="baseSize">
                    <size>
                     <width>92</width>
                     <height>92</height>
                    </size>
                   </property>
                   <layout class="QGridLayout" name="gridLayout_8">
                    <property name="leftMargin">
                     <number>0</number>
                    </property>
                    <property name="topMargin">
                     <number>0</number>
                    </property>
                    <property name="rightMargin">
                     <number>0</number>
                    </property>
                    <property name="bottomMargin">
                     <number>0</number>
                    </property>
                    <property name="spacing">
                     <number>0</number>
                    </property>
                    <item row="0" column="0">
                     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
                      <item>
                       <widget class="QLabel" name="podcastCover">
                        <property name="minimumSize">
                         <size>
                          <width>128</width>
                          <height>128</height>
                         </size>
                        </property>
                        <property name="text">
                         <string notr="true"/>
                        </property>
                        <property name="pixmap">
                         <pixmap resource="../images.qrc">:/img/no-cover.svg</pixmap>
                        </property>
                        <property name="scaledContents">
                         <bool>false</bool>
                        </property>
                       </widget>
                      </item>
                      <item>
                       <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,0,1">
                        <property name="spacing">
                         <number>2</number>
                        </property>
                        <property name="leftMargin">
                         <number>9</number>
                        </property>
                        <item>
                         <widget class="QLabel" name="podcastTitle">
                          <property name="font">
                           <font>
                            <pointsize>12</pointsize>
                            <weight>75</weight>
                            <bold>true</bold>
                           </font>
                          </property>
                          <property name="text">
                           <string>Podcasts</string>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLabel" name="podcastWeb">
                          <property name="font">
                           <font>
                            <pointsize>10</pointsize>
                           </font>
                          </property>
                          <property name="text">
                           <string>Web</string>
                          </property>
                          <property name="openExternalLinks">
                           <bool>true</bool>
                          </property>
                         </widget>
                        </item>
                        <item>
                         <widget class="QLabel" name="podcastDesc">
                          <property name="font">
                           <font>
                            <pointsize>10</pointsize>
                           </font>
                          </property>
                          <property name="text">
                           <string>Description</string>
                          </property>
                          <property name="alignment">
                           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                          </property>
                          <property name="wordWrap">
                           <bool>true</bool>
                          </property>
                          <property name="openExternalLinks">
                           <bool>true</bool>
                          </property>
                         </widget>
                        </item>
                       </layout>
                      </item>
                     </layout>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QTableWidget" name="episodesTable">
                   <property name="font">
                    <font>
                     <pointsize>9</pointsize>
                    </font>
                   </property>
                   <property name="contextMenuPolicy">
                    <enum>Qt::CustomContextMenu</enum>
                   </property>
                   <property name="editTriggers">
                    <set>QAbstractItemView::NoEditTriggers</set>
                   </property>
                   <property name="showDropIndicator" stdset="0">
                    <bool>false</bool>
                   </property>
                   <property name="dragDropOverwriteMode">
                    <bool>false</bool>
                   </property>
                   <property name="selectionMode">
                    <enum>QAbstractItemView::ExtendedSelection</enum>
                   </property>
                   <property name="selectionBehavior">
                    <enum>QAbstractItemView::SelectRows</enum>
                   </property>
                   <attribute name="horizontalHeaderVisible">
                    <bool>false</bool>
                   </attribute>
                   <attribute name="verticalHeaderVisible">
                    <bool>false</bool>
                   </attribute>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Queue</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2" rowstretch="1" columnstretch="1">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QGridLayout" name="gridLayout_7">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <property name="spacing">
              <number>0</number>
             </property>
             <item row="0" column="0">
              <widget class="QSplitter" name="splitter_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <widget class="QListWidget" name="queueList">
                <property name="editTriggers">
                 <set>QAbstractItemView::NoEditTriggers</set>
                </property>
                <property name="showDropIndicator" stdset="0">
                 <bool>false</bool>
                </property>
                <property name="dragDropMode">
                 <enum>QAbstractItemView::NoDragDrop</enum>
                </property>
                <property name="selectionMode">
                 <enum>QAbstractItemView::ExtendedSelection</enum>
                </property>
                <property name="selectionBehavior">
                 <enum>QAbstractItemView::SelectRows</enum>
                </property>
               </widget>
               <widget class="QTextBrowser" name="infoEpisodeLabel"/>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>Downloads</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <widget class="QListWidget" name="downloadsList"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../images.qrc"/>
  <include location="../../../../.designer/images.qrc"/>
 </resources>
 <connections/>
</ui>
